docs/devGuide/development/migratingToTypeScript.md

```diff
@@ -95,6 +95,31 @@ We have decided on structuring the commits as two commits: a "Rename" commit and
 
 You are now ready to create a pull request for the changes to the repository.
 
+### (Optional) Step 4: Changing the commit
+
+While refactoring you may encounter scenarios where you might have to amend your code after a commit. There are 2 common scenarios which we will cover. 
+
+##### Scenario 1 - you have 2 commits and want to add on to the second commit: 
+
+To append a new commit onto the already existing commit you can do the following: 
+1. Make the changes you want to add to the second commit
+2. Stage the changes using `git add`
+3. Run `git commit --amend`
+4. This will open your default text editor, where you can edit the commit message for the second commit to reflect the new changes you have made
+5. Save and close the text editor. This will replace the second commit with the updated changes.
+
+<box type="warning">
+It's important to note that amending a commit changes its commit hash, which means that if you've already pushed the original commit to a remote repository, you'll need to force push your changes.
+</box>
+
+##### Scenario 2 - you have 3 commits and want to make it into 2 commits:
+
+1. Run `git rebase -i HEAD~3` to start an interactive rebase session. This will show a list of the three most recent commits in your text editor.
```

Comment:
```suggestion
1. Run `git rebase -i master` to start an interactive rebase session. This will show a list of the recent commits in your text editor.
```

Rebasing on master can allow this command to be used regardless of how many commits there are! Also bonus of making the rebase commit (for merging into master) easier later on.